src/PythonMigrationViewExtension/PythonMigrationViewExtension.cs

```diff
@@ -133,7 +133,7 @@ private void OnCurrentWorkspaceChanged(IWorkspaceModel workspace)
         {
             NotificationTracker.Remove(CurrentWorkspace.Guid);
             CurrentWorkspace = workspace as WorkspaceModel;
-            if (Configuration.DebugModes.IsEnabled("Python3DebugMode") && PythonDependencies.ContainsIronPythonDependencies())
+            if (Configuration.DebugModes.IsEnabled("Python3DebugMode") && !Models.DynamoModel.IsTestMode && PythonDependencies.ContainsIronPythonDependencies())
```

Comment:
NIT: do you mind making this into 2 lines?

Reply:
@mjkkirschner Done!